### PR TITLE
added skyfield-data

### PR DESCRIPTION
**Pull request by Matt-BF**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/skyfield-data/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2019 Bruno Bord, PeopleDoc Inc.
```
Comment (beckermr) [inline recipes/skyfield-data/LICENSE.txt]: Please make a PR to include this license file in the upstram sdist via the MANIFEST.in.

File: recipes/skyfield-data/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    
+test:
+  imports:
+    - skyfield_data
+    
+about:
+  home: https://github.com/brunobord/skyfield-data
+  license: MIT
+  license_family: MIT
+  license_file:
```
Comment (beckermr) [inline recipes/skyfield-data/meta.yaml]: You need to specify the path to the license file here. Putting `LICENSE.txt` as the entry here should be sufficient.

File: recipes/skyfield-data/LICENSE.txt (hunk 1 above)
Comment (Matt-BF) [reply recipes/skyfield-data/LICENSE.txt]: I'm sorry, I've never messed with MANIFEST.in files, so I'm not sure how I would go about to do this

File: recipes/skyfield-data/meta.yaml (hunk 2 above)
Comment (Matt-BF) [reply recipes/skyfield-data/meta.yaml]: Done!

File: recipes/skyfield-data/LICENSE.txt (hunk 1 above)
Comment (beckermr) [reply recipes/skyfield-data/LICENSE.txt]: No worries. Make an issue on the upstream repo and I will merge now.

**Pull request merged**